lib/pronto/cli.rb

```diff
@@ -55,7 +60,7 @@ def run(path = '.')
 
       formatters = ::Pronto::Formatter.get(options[:formatters])
 
-      commit_options = %i[staged unstaged index]
+      commit_options = %i[staged unstaged index workdir]
```

Comment:
Shouldn't we look at this first, as this option kind of conflicts with the other two options?

Reply:
Ah, meaning bump it up to the beginning of the list? Yeah, that probably makes sense. I'll update it.